tests/Integration/Consumer/ConsumerTest.php

```diff
@@ -68,6 +68,25 @@ public function test_it_should_throw_timeout_with_multiple()
         $consumer->callback($callback)->consume(1);
     }
 
+    /**
+     * @requires extension pcntl
+     * @
+     */
+    public function test_it_should_handle_sigterm_signal()
```

Comment:
`signalHandler` can deal with three kinds of signals but just sigterm is beeing tested.

Reply:
Yaeh, i've dont made it cause `signalHandler` has a simple switch so i dont want to test the switch itself, only the handling of signal